wurst/_items/ScrollOfCyclone.wurst

```diff
@@ -0,0 +1,20 @@
+package ScrollOfCyclone
+
+import ClosureEvents
+import ClosureTimers
+
+let ABILITY_SCROLL_OF_CYCLONE = 'S000'
+let BUFF_CYCLONE = 'Bcyc'
```

Comment:
Do buff IDs normally have lowercase? Seems odd that it's all uppercase most of the time but not here.